docs/plugins/videoplayer.md

```diff
@@ -377,40 +375,51 @@ VideoPlayer.visible // false
 ## Events
 
 The VideoPlayer plugin emits a number of [media events](https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Media_events) to its
-consumer (as specified in [`VideoPlayer.consumer()`](#consumer)).
+*consumer* (which is specified in the [`VideoPlayer.consumer()`](#consumer) method).
```

Comment:
maybe it should be "specified via the consumer method" ?

Reply:
Agreed.